package-lock.json

```diff
@@ -7180,28 +7180,28 @@
       "dependencies": {
```

Comment:
Let's avoid checking in no-op'y changes to this file unless it's a specific PR dedicated to that.